README.md

```diff
@@ -1,2 +1,130 @@
-# plugin-grunt-tasks
-Several tasks requires for our plugins
+# grunt-yoast-tasks
+
+> the plugin that contains all custom yoast tasks
```

Comment:
Please capitalize the first word of the sentence and the name Yoast.

Reply:
okey, but other [grunt](https://gruntjs.com/plugins) plugins don't do this. For example : https://github.com/gruntjs/grunt-contrib-jshint